src/main/java/org/zalando/nakadi/repository/kafka/KafkaLocationManager.java

```diff
@@ -108,6 +109,7 @@ public Properties getKafkaProducerProperties() {
         producerProps.put("request.timeout.ms", kafkaSettings.getRequestTimeoutMs());
         producerProps.put("batch.size", kafkaSettings.getBatchSize());
         producerProps.put("linger.ms", kafkaSettings.getLingerMs());
+        producerProps.put(ProducerConfig.COMPRESSION_TYPE_CONFIG, "lz4");
```

Comment:
@lmontrieux cda0b7ab
About moving it to KafkaSettings: I think it is not going to change from env to env(at least I do not see it that way). There are also other properties, which are not in that config.